README.md

```diff
@@ -2196,6 +2196,52 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+- <a id='wrap-single-line-declaration-bodies'></a>(<a href='#wrap-single-line-declaration-bodies'>link</a>) **Wrap single-line function, init, subscript, and computed property bodies onto multiple lines.**
```

Comment:
Wording like this might be more "style advice" oriented as opposed to imperative "code formatter" oriented. 

```suggestion
- <a id='wrap-single-line-declaration-bodies'></a>(<a href='#wrap-single-line-declaration-bodies'>link</a>) **Always wrap function and property bodies onto multiple lines.**
```